CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Change log
+
+## 0.1.0 (2021-11-29)
+
+* Initial release of ```s3_file_downloader```
```

Comment:
Only one set of quotes is enough

`s3_file_downloader`
![image](https://user-images.githubusercontent.com/668524/144014264-83f53418-431a-4d2b-8969-7f36f4123af0.png)


Also, you have still not released any version, so it will be not `## 0.1.0 (2021-11-29)`, but `master (unreleased)

Like [here](https://github.com/ONLYOFFICE-QA/onlyoffice_bugzilla_helper/blob/master/CHANGELOG.md#master-unreleased) for example